yamlgen/Cargo.toml

```diff
@@ -8,4 +8,4 @@ license = "MIT OR Apache-2.0"
 [build-dependencies]
 kube = { version = "0.74", default-features = false }
 model = { version = "0.0.2", path = "../model" }
-serde_yaml = "0"
+serde_yaml = "0.8"
```

Comment:
We shouldn't need to do this unless we want to pin to the `0.8` minor version. By specifying the major version only, we get minor version bumps when we do `cargo update`.

Reply:
This is because there are breaking changes in 0.9.0 of serde-yaml: https://github.com/bottlerocket-os/bottlerocket-test-system/issues/559

and it didn't make sense to address that in this PR